CVE Search v2 - Fixed an issue where cvss score is null (#16412)
demisto/content: Packs/CVESearch/ReleaseNotes/1_0_11.md

#### Integrations
##### CVE Search v2

> - Fixed an issue where the CVSS score is null.

Reviewer: ```suggestion
- Fixed an issue where a null CVSS score was handled poorly.
```

Author: done
